### PR TITLE
Move some peerDependencies to dependencies

### PR DESCRIPTION
**Pull request by NameFILIP**
See https://github.com/uber/react-digraph/issues/58 + a couple of other minor things (lock file, rename react lifecycle method)

### CODE REVIEW

**Comment by NameFILIP** [comment]
@ajbogh trying to squeeze in a couple of other minor changes. Let me know if I should extract that into a separate PR

File: src/components/graph-view.js [hunk 1]
```diff
@@ -199,7 +199,7 @@ class GraphView extends Component {
       .on('keydown', null);
   }
 
-  componentWillReceiveProps(nextProps) {
+  UNSAFE_componentWillReceiveProps(nextProps) {
```
Comment (ajbogh) [inline src/components/graph-view.js]: Don't change this yet. While we are working toward supporting 16.3 natively, we still run on React 15 in the 4.x versions. The tests will use React 16 to make sure we're compatible with newer projects.

The next major version will require React 16.3+

**Comment by NameFILIP** [comment]
@ajbogh Updated according to your comment

**Comment by NameFILIP** [comment]
@ajbogh does everything look good from my end?

**Comment by ajbogh** [comment]
Looks good. Sorry about the delay, I was on vacation. Will merge and publish on Monday!

**Comment by ajbogh** [comment]
I've tested this and it works. I did notice that the example site no longer pans or zooms, but that's unrelated to this change.

**Comment by ajbogh** [comment]
I don't have permission to publish the change in NPM so I've asked @johnvf to do that. The new tag is already in git though.

**Pull request merged**